docs/configuration/config-file-reference.md

```diff
@@ -846,9 +846,8 @@ ruler_client:
 # CLI flag: -ruler.evaluation-interval
 [evaluation_interval: <duration> | default = 1m]
 
-# Duration to delay the evaluation of rules to ensure they underlying metrics
-# have been pushed to cortex.
-# CLI flag: -ruler.evaluation-delay-duration
+# Deprecated. Please use -ruler.evaluation-delay-duration instead.
+# CLI flag: -ruler.evaluation-delay-duration-deprecated
```

Comment:
Hrm, this deprecated notice is not super useful for the YAML config. Hrm. Not sure how to fix it though.

Reply:
This flag only exists to have YAML field with "deprecated" documentation. Ideally we would hide CLI flag, but our tooling is not there yet. Given that this will be removed in Cortex 1.6, let's not waste too much time on it.